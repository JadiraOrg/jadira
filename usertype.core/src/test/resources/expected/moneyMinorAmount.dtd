<!ELEMENT dataset (
    MONEYMINORAMOUNT*)>

<!ELEMENT MONEYMINORAMOUNT EMPTY>
<!ATTLIST MONEYMINORAMOUNT
    ID CDATA #REQUIRED
    MONEY CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

