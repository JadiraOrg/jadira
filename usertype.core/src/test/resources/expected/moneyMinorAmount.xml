<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "moneyMinorAmount.dtd">
<dataset>
  <moneyMinorAmount ID="0" MONEY="10000" NAME="test_0"/>
  <moneyMinorAmount ID="1" MONEY="10010" NAME="test_1"/>
  <moneyMinorAmount ID="2" MONEY="99" NAME="test_2"/>
  <moneyMinorAmount ID="3" NAME="test_3"/>
</dataset>